### PR TITLE
Replace Cobertura with JaCoCo

### PR DESCRIPTION
**Pull request by Plat251**
Both Cobertura and JaCoCo are code reporting tools, but JaCoCo isn't abandoned and supports JDK 11. Cobertura breaks JDK 11, that's why we had those Travis builds fail in the past.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -27,17 +43,29 @@
...
-                        <format>html</format>
-                        <format>xml</format>
-                    </formats>
-                    <check/>
-                </configuration>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>2.22.1</version>
             </plugin>
+			<plugin>
```
Comment (blackk100) [inline pom.xml]: Extra indentation in this block. It's fine otherwise.

Comment (Plat251) [reply pom.xml]: Fixed in 741d693

**Comment by Veradux** [comment]
Actually, why do we care about jdk 11? Didn't we agree that it will be only jdk 8.

**Comment by Plat251** [comment]
That's a good question. It's only one of the benefits of this change.
Second one is that JaCoCo isn't actually abandoned, unlike [Cobertura](https://github.com/mojohaus/cobertura-maven-plugin/issues/43).

This PR doesn't affect a significant amount of things, but we will be using a plugin that's actual, up-to-date, and works with JDK 11 as a cherry on top.

**Pull request merged**